### ISSUE TITLE
on version 1.0.0-beta.41, `web3.utils.randomHex` is undefined

### ISSUE THREAD

**Issue by BigMurry** [ethereum/web3.js#2270]
#### Expected behavior
`web3.utils.randomHex` should defined and work

#### Actual behavior
`web3.utils.randomHex` is not a function. It's undefined.

#### Steps to reproduce the behavior
```javascript
import Web3 from 'web3';
const web3 = new Web3('http://localhost:8545');
console.log(typeof web3.utils.randomHex);
```

#### Error Logs
`TypeError: web3.utils.randomHex is not a function`

#### Versions
Web3.js - 1.0.0-beta.41

**Comment by caffeinum** [comment]
Can confirm this bug, OS X, NodeJS 10.14

Rolled back to beta37, it works OK there.

```bash
npm i web3@1.0.0-beta.37
```

Edit: on beta37, there is another bug: ethereum/web3.js/issues/2077, so rolling back to beta35. Nevermind Lambo, when stable release 😁

**Comment by nivida** [comment]
Thanks for submitting this issue! I'll fix this issue on Monday.

**Comment by nivida** [comment]
I'm closing this issue because it got already reported and fixed in #2238
